bigframes/geopandas/geoseries.py

```diff
@@ -67,6 +68,14 @@ def area(self, crs=None) -> bigframes.series.Series:  # type: ignore
             f"GeoSeries.area is not supported. Use bigframes.bigquery.st_area(series), instead. {constants.FEEDBACK_LINK}"
         )
 
+    @classmethod
+    def from_wkt(
+        cls, data, index=None, crs=None, on_invalid="raise", **kwargs
```

Comment:
Since we don't support `crs` or `on_invalid` arguments, either delete them here and in the superclass or raise an error if they pass in something we can't support.